sdk/storage/Azure.Storage.Common/src/Shared/StorageResponseClassifier.cs

```diff
@@ -42,19 +42,18 @@ public override bool IsRetriableResponse(HttpMessage message)
                 }
             }
 
-            // https://github.com/Azure/azure-sdk-for-net/issues/44324
-            //// Retry select Copy Source Error Codes.
-            //if (message.Response.Status >= 400 &&
-            //    message.Response.Headers.TryGetValue(Constants.HeaderNames.CopySourceErrorCode, out string copySourceError))
-            //{
-            //    switch (copySourceError)
-            //    {
-            //        case Constants.ErrorCodes.InternalError:
-            //        case Constants.ErrorCodes.OperationTimedOut:
-            //        case Constants.ErrorCodes.ServerBusy:
-            //            return true;
-            //    }
-            //}
+            // Retry select Copy Source Error Codes.
+            if (message.Response.Status >= 400 &&
+                message.Response.Headers.TryGetValue(Constants.HeaderNames.CopySourceErrorCode, out string copySourceError))
+            {
+                switch (copySourceError)
+                {
+                    case Constants.ErrorCodes.InternalError:
```

Comment:
We may need to modify this, depending on the outcome of our project around redoing SDK retries.